PhysicsTools/MXNet/test/test_mxnet.py

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env python
+import mxnet as mx
+import numpy
+a = mx.nd.ones((2, 3))
+b = a * 2 + 1
+b.asnumpy()
+numpy.array([[ 3.,  3.,  3.],
+       [ 3.,  3.,  3.]], dtype=numpy.float32)
```

Comment:
I do not understand these last 2 lines. I think just copy pasted the example. These lines are generated by `b.asnumpy()`

Reply:
I didn't understand either until I finally read the text above it - it was a code so I took it as part of the example 😅